src/style/App.css

```diff
@@ -1,8 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&family=Stylish&display=swap');
+
+*  {
+  
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Roboto',
+  sans-serif;
+}
+:root {
+  font-size: 62.5%;
+}
+body {
+  font-family: sans-serif;
+  font-size: 1.6rem;
+  background-image: url();
+  background: rgb(179, 129, 213);
+  background: linear-gradient(90deg, rgba(179, 129, 213, 1) 8%, rgba(29, 199, 253, 1) 50%, rgba(69, 252, 210, 1) 100%);
+
+}
+.App {
+  margin: 3rem 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.Title {
+  font-family: 'Stylish',
+  sans-serif;
```

Comment:
Could you review the format please? Sometimes you have spaces after each declaration and sometimes not, could we be consistent with just one? I like spaces, but you can decided.

Reply:
done!